extension/packages/nextjs/app/page.tsx.args.mjs

```diff
@@ -0,0 +1,27 @@
+// CHALLENGE-TODO: Update the title, description, and challengeDeliverable to reflect your challenge
+export const description = `
+  <div className="flex items-center flex-col flex-grow mt-4">
+    <div className="px-5 w-[90%]">
+      <h1 className="text-center mb-6">
+        <span className="block text-4xl font-bold">{challengeTitle}</span>
```

Comment:
Maybe here `{challengeTitle}` I think lets have the String as CHALLENGE TITLE

otherwise it breaks : 

<img width="1258" alt="Screenshot 2025-03-19 at 5 25 01 PM" src="https://github.com/user-attachments/assets/b602494f-6ee9-4f90-a303-98a9c8c4ab81" />

Reply:
I changed this.